apps/i18n/common/en_us.json

```diff
@@ -2869,6 +2869,13 @@
   "progressV2_feedback_shareMoreLinkText": "Share more feedback",
   "progressV2_feedback_thumbsUp": "I like it!",
   "progressV2_feedback_thumbsDown": "I don't like it",
+  "submitProjectGallery_alreadySubmitted": "Submitted to Featured Projects gallery",
```

Comment:
Should we end the sentence with a `.`?

Reply:
This will be a button label.

Reply:
Actually this is displayed text. After discussion, landing on:
'Thanks for your submission.'

cc: @samantha-code